xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java

```diff
@@ -27,6 +27,7 @@
   protected static final String USE_HIGHER_UPDATE_FREQUENCY =
       "Binance_Orderbook_Use_Higher_Frequency";
   protected static final String USE_REALTIME_BOOK_TICKER = "Binance_Ticker_Use_Realtime";
+  protected static final String USE_MAX_FETCH_ORDER_BOOK_LIMIT = "Binance_Max_Fetch_Order_Book_Limit";
```

Comment:
the words "use" and "max" make little sense for this parameter

just name it

FETCH_ORDER_BOOK_LIMIT = "Binance_Fetch_Order_Book_Limit"

Reply:
Does this mean that this parameter will no longer be Boolean, but will become an Integer with a specific limit value? Do I need to validate it for valid values (5, 10, 20, 50, 100, 500, 1000, 5000) in this case, or is the correctness of the value the responsibility of the api user?

Reply:
@walec51 to speed up the acceptance of the PR, decided to redo it before your answer. Renamed parameter name and used only if integer inside parameter.